pysite/base_route.py

```diff
@@ -81,11 +87,37 @@ def error(self, error_code: ErrorCodes):
         return response
 
 
+class DBViewMixin:
```

Comment:
class docstrings would be nice too

Reply:
and yes, I get that it's because these things aren't really public classes and methods, but for the sake of this being a learning project for many of the contributors and eventually also onlookers, it would be nice if we treated all of it as if it were public and just docstringed everything.